src/invidious/comments.cr

```diff
@@ -675,6 +689,27 @@ def content_to_comment_html(content, video_id : String? = "")
     text = "<s>#{text}</s>" if run["strikethrough"]?
     text = "<i>#{text}</i>" if run["italics"]?
 
+    # check for custom emojis
+    if run["emoji"]?
+      if run["emoji"]["isCustomEmoji"]?.try &.as_bool
+        if emojiImage = run.dig?("emoji", "image")
+          emojiAlt = emojiImage.dig?("accessibility", "accessibilityData", "label").try &.as_s || text
+          emojiThumb = emojiImage["thumbnails"][0]
+          text = String.build do |str|
+            str << %(<img alt=") << emojiAlt << "\" "
+            str << %(src="/ggpht) << URI.parse(emojiThumb["url"].as_s).request_target << "\" "
+            str << %(title=") << emojiAlt << "\" "
+            str << %(width=") << emojiThumb["width"] << "\" "
+            str << %(height=") << emojiThumb["height"] << "\" "
+            str << %(style="margin-right:2px;margin-left:2px;"/>)
```

Comment:
Can you create a dedicated class (e.g `channel-emoji`) ad put the styles in `default.css`?
I'd like to see all inline CSS to be removed in the future, so it's better if we can start now ^^